### ISSUE TITLE
I can't install CeBuilder on my iMac [my-product] - 

### ISSUE THREAD

**Issue by miguelitolaparra** [inspireui/support#1767]
> - Thank you for reaching please follow the format below to create a new ticket, please explain as more detail as possible and allow 24 - 48 hours to receive an answer to your ticket.
> - Try to explain your all problems in one ticket, instead of creating multi tickets.
> - After creating an issue, please take the screenshot of http://codecanyon.net/download - to let us know you have purchased our product, or quick note as a comment that you have created this ticket, example: https://tppr.me/DJA8L
> - IMPORTANT: We will close the ticket if you could not prove as our customer.

---------

1. My issue:
Hi, I have downloaded the CeBuilder installation file from your site but my iMac does not allow me to install it.
I show you two screenshots of the application.
It basically tells me the following:
"CeBuilder.app cannot be opened because Apple cannot check for malicious software.
You must update this software. Contact the developer for more information. "

I don't know what else I can do to help me with the installation problem.
I hope with this you can help me and thus facilitate the configuration of my application
PROBLEMS WITH THE WORDPRESS SITE
To explain everything in a single ticket, I must tell you that I cannot download the trial WordPress site to create the application.
Can you help me with this?
Can you tell me the steps to follow to configure WordPress?
I have a domain available, where do I download the necessary tools to configure WordPress?
And if I don't have a domain, how do I set up the WordPress site?
Greetings
I show you the screenshots that I have thought necessary, if you need more sizes, let me know
2. Screenshots issues:
<img width="1577" alt="purchase" src="https://user-images.githubusercontent.com/28591454/71821113-894ae700-3091-11ea-9f70-a8eb78b2579b.png">

<img width="760" alt="errorCeBuilder" src="https://user-images.githubusercontent.com/28591454/71821205-ca42fb80-3091-11ea-97ee-1804cfa11a95.png">

<img width="995" alt="Captura de pantalla 2019-12-31 a las 15 19 43" src="https://user-images.githubusercontent.com/28591454/71821227-ddee6200-3091-11ea-9f54-1e8827b42642.png">

3. Proof of my purchased:

<img width="1577" alt="purchase" src="https://user-images.githubusercontent.com/28591454/71821266-f3638c00-3091-11ea-8d0e-214fc5eaeaff.png">

3. Config files (optional)

**Comment by minhcasi** [comment]
@miguelitolaparra - could you try this similar solution to open the cebuilder (it seems related to the permission on MacOS) - https://www.minicreo.com/mac-uninstaller/app-cannot-be-opened-on-macos-catalina.html

**Comment by minhcasi** [comment]
The pre-config site could be download from - https://tppr.me/cNAmp - thanks

**Comment by miguelitolaparra** [comment]
>  could you try this similar solution to open the cebuilder (it seems related to the permission on MacOS) - https://www.minicreo.com/mac-uninstaller/app-cannot-be-opened-on-macos-catalina.html

[Solved] App Can't Be Opened Because Apple Cannot Check It for Malicious Software | MiniCreo
[Common macOS Catalina Problems] Trying to install Skype, Adobe Creative Cloud, Tor Browser or other apps on macOS Catalina but get a message saying XXX can't be opened because Apple cannot check it for malicious software. Here are quick fixes.Thanks for your support.
I used the tool Omni Remover taught me, but when I tried to install CeBuilder again, it gave me the following message: "CeBuilder.app will damage your computer. You should move it to the trash."
Of course I have moved it to the bin.
How can I correct this error?
How does Omni Remover work?
Thank you

**Comment by miguelitolaparra** [comment]
Thanks for your support. @minhcasi 
I used the tool Omni Remover taught me, but when I tried to install CeBuilder again, it gave me the following message: "CeBuilder.app will damage your computer. You should move it to the trash."
Of course I have moved it to the bin.
How can I correct this error?
How does Omni Remover work?
Thank you

<img width="760" alt="errorDeNuevo" src="https://user-images.githubusercontent.com/28591454/72078050-a6cdba00-32f8-11ea-847c-93d4aeac14a1.png">

**Comment by minhcasi** [comment]
stop support

**Comment by miguelitolaparra** [comment]
To close the aid you are faster than to really help people. You really are unprofessional. They must recognize their mistakes

**Comment by miguelitolaparra** [comment]
To close the aid you are faster than to really help people. You really are
unprofessional. They must recognize their mistakes

**Comment by Varun24041993** [comment]
Hi I faced similar problem in my Mac but I have found a way around instead of downloading 0.38.0 version, try downloading 0.37.0. After downloading it copy the file in the application folder using ifinder, then keep the control key(not command key) pressed and click on the installer (.dmg) file then select open from the list. I have attached screenshots both for the cebuilder version to download and the Mac version I have installed on my pc.
![Screenshot_1](https://user-images.githubusercontent.com/56767455/72781190-af9e8400-3c46-11ea-921a-4834d0c53cf8.png)
![Screenshot_2](https://user-images.githubusercontent.com/56767455/72781204-b88f5580-3c46-11ea-90ea-37348b1b135d.png)
